src/live_restore/live_restore_fs.c

```diff
@@ -1155,27 +1145,38 @@ __wt_live_restore_metadata_to_fh(
     __wt_readlock(session, &lr_fh->bitmap_lock);
     lr_fh->allocsize = lr_fh_meta->allocsize;
     /*
-     * Only allocate a bitmap for a file that has been newly created in the destination and has a
-     * backing source file.
+     * !!!
+     * While the live restore is in progress the bit count reported by in the live restore meta can
+     * hold three states:
+     *  (0)     : This means file has not yet had a bitmap written to the metadata file and
+     *            therefore no relevant writes have gone to the destination.
```

Comment:
What is a relevant write? Is it possible to have an irrelevant write?

Reply:
Yes, I will elaborate a bit here.